docs/guides/initialize-account.mdx

```diff
@@ -0,0 +1,191 @@
+2. Optional: Find the instance you want to use from the [Instances](https://quantum.cloud.ibm.com/instances) page. Click the icon to copy its CRN, then save it in a secure location so you can use it to identify the instance.
+
+
+## Connect Qiskit with your Qiskit Runtime service instance
+
+<Admonition type="note">
+These instructions are designed for `qiskit_ibm_runtime` v0.42 or later. Some features are not enabled in earlier versions of `qiskit_ibm_runtime`. In these cases, you should always provide a value for `channel`, `token`, and `instance`, either explicitly or through a saved account.
+</Admonition>
+
+The basic code to connect Qiskit with your Qiskit Runtime service instance follows.  However, there are several different ways to customize the `QiskitRuntimeService` options, depending on your needs.  These options are described in the following sections.
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+service = QiskitRuntimeService(channel=<channel>, token=<cloud_api_key>,
+        instance=<instance_CRN>)
+```
+
+<Admonition type="note">
+The default channel is `ibm_quantum_platform`.  Because this is almost always the appropriate channel, it is not included in the examples.
+</Admonition>
+
+<span id="explicit"></span>
+### Quick start path: Specify credentials explicitly
+
+The fastest way to get a `QiskitRuntimeService` instance running is the direct instantiation method: explicitly provide the API token (key) and CRN (instance identifier) every time you need to initialize the Qiskit Runtime service. See [Find your access credentials](#access-credentials) if necessary.
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+service = QiskitRuntimeService(token=<cloud_api_key>, instance=<instance_CRN>)
+```
+
+This path is reliable but can be tedious if you need to load the same details repeatedly. To avoid loading your credentials multiple times, if you are working in a trusted Python environment (such as on a personal laptop or workstation), you can use saved account credentials, as described in the following section.
+
+Although the `instance` input parameter is optional, it is recommended that you always provide this information, unless you want to use a single service to work with several instances. In this situation, see the [automatic instance selection](#auto-select) section.
+
+<span id="saved"></span>
+### Quick start path: Specify saved credentials
+
+If you have already saved your credentials, use the following code to use your default credentials. For instructions to save credentials, see [Save your access credentials.](/docs/guides/save-credentials)
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+# run every time you need the service
+service = QiskitRuntimeService()
+...
+```
+
+If you named one or more sets of credentials, for example, for open and premium access, use the following code to use a set of named credentials.
```

Comment:
I know we try to minimize the noise of parens but I really want to put (for example, for open and premium access) in parentheses. Maybe there's another creative way to do it....

Reply:
maybe dashes? 

 - for example, for open and premium access -